app/lib/features/chat/providers/notifiers/chat_room_notifier.dart

```diff
@@ -429,6 +434,17 @@ class ChatRoomNotifier extends StateNotifier<ChatRoomState> {
     String eventId = eventItem.uniqueId();
 
     String? inReplyTo = eventItem.inReplyTo();
+
+    // user read receipts for timeline event item
```

Comment:
I am unclear about these receipt changes in here ... are they meant to be in this PR? What do they relate to in the description?